frontend/app/src/pages/main/events/index.tsx

```diff
@@ -123,6 +123,16 @@ function EventsTable() {
     return filter?.value as Array<string>;
   }, [columnFilters]);
 
+  const workflows = useMemo(() => {
+    const filter = columnFilters.find((filter) => filter.id === 'workflows');
+
+    if (!filter) {
+      return;
+    }
+
+    return filter?.value as Array<string>;
+  }, [columnFilters]);
+
```

Comment:
Also not sure if this gets too repetitive as I just copied it from above

Reply:
Honestly seems fine, it's pretty readable